crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -600,6 +608,8 @@ fn spawn_background_processor(
                     false
                 })
             },
+            // TODO: think about how to make this configurable per platform
+            true,
```

Comment:
❓ what is this 😅

Reply:
c136039594517a135d3572be340d75bd5608f95c